### ISSUE TITLE
Cant Edit Product Attributes!

### ISSUE THREAD

**Issue by justdudeone** [hyyan/woo-poly-integration#374]
Hello!
After I create many attributes and values, I create a new product with the them – all works fine. But if then I trying to edit this product – I can edit text&announce, but cannot edit attributes-values and all other parameters at ProductData panel! I see them, but they all are disabled!
I already try to disable all WordPress plugins except 3: WooCommerce / Polylang / HyyanWooCommercePolylangIntegration – because those plugins a base-core and critical for site working – bug is still exist and spoil all work! Problem solves only after I turn off ‘Hyyan WooCommerce Polylang Integration’! But this is one of 3 core plugins of my website! :(((
Please, help!

**Comment by Jon007** [comment]
The “Fields Locker” disables fields which are synchronized when you try to edit in the translated versions of the product. This is because the edits will be overwritten from the shop base language version of the product.

The “Fields Locker” can be turned off in admin menu Settings, WooPoly, Features, (/wp-admin/options-general.php?page=hyyan-wpi Features tab, but this isn’t a great idea.

You can control which fields are synchronised (and therefore disabled except in base language) admin menu Settings, WooPoly, Metas List, (/wp-admin/options-general.php?page=hyyan-wpi Metas List tab).

All these settings are also explained in the documentation wiki:
https://github.com/hyyan/woo-poly-integration/wiki

**Comment by justdudeone** [comment]
Hi, thanks for reply.

I also think as you wrote:
"The “Fields Locker” disables fields which are synchronized <em>when you try to edit in the translated versions</em> of the product. This is because the edits will be overwritten from the shop base language version of the product."

But, at this thread, I have ONLY ONE LANGUAGE product, that does NOT synchronized with any other product at all! But... disabling this option are really helps - attributes becomes editable. Is it a bug?

PS All those small nuances just endure the nerves and waste a ton of time. :(

**Comment by Jon007** [comment]
What is the shop base language?
What is the language of the untranslated product?
Again note that you can turn the fields locker off in Settings, Woopoly, Features

**Comment by justdudeone** [comment]
WP admin CP lng = Ru.
Polylang languages = English (order=0, set as default), Russian (order=1).
Products that has blocked for editing attributes = Ru.
This products does NOT have a translated version of product (Eng).
WooPoly setting "Fields Locker" = was Enabled (checked) - product attributes was not editable, when this setting are disabled - product attributes are normally editable.

**Comment by Jon007** [comment]
Sure this is expected, as per the documentation products should be edited in shop base language.
Also note please don’t change base language, especially if you have already created variations.

**Comment by justdudeone** [comment]
Thanks for explanation. 
But I really dont see any sense for this complication. 
1) Why products can't be firstly created at any other language and then translated to any other language - so the first created language automatically becomes for this product as "main-basic"?
2) Why FieldsLocker generally needs "main-basic" product language??? Why it just can not work with all linked products equally? For example I create product firstly at Eng language, then my partners at other countries translate this product to their languages: Fr/De. Maybe instantly or maybe later one of them finds some mistake at attributes, for example De-partner. So it will be very useful if this partner by itself can correct-change this problem-mistake for all linked products! So the best way is to hard link all attributes for all translated products without locking of they editing! Does WooPoly can do this?
(Excuse if my questions are too easy, I just totally overloaded and also trying to discover WP-WooC from the beginning.)

**Comment by Jon007** [comment]
Polylang doesn’t implement permissions by language so the DE editor would also be able to correct product data in the original language.
Actually I think it could work as you suggest for simple products, but not for variable products due to the way this plugin links product variations-  I had already opened a separate issue to propose improvements to this
